packages/ui-kit/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,46 @@
+import { VFC, ReactNode, ButtonHTMLAttributes } from "react";
+import clsx from "clsx";
+
+import styles from "./Button.module.css";
+import { Text } from "../Text";
+
+interface ButtonLabelProps {
+  content: string;
+  className?: string;
```

Comment:
same

Reply:
`classNames` is not necessary here, just classic old `className`